src/components/organisms_new/StartPage/StartPage.tsx

```diff
@@ -1,10 +1,13 @@
 import {useEffect, useState} from 'react';
 
+import {WALK_THROUGH_STEPS} from '@constants/walkthrough';
```

Comment:
We should add this inside `shared` folder as most probably `@constants` will be removed in the near future